src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -121,12 +124,12 @@ class EntityClient (requestContext: RequestContext)(implicit protected val execu
       importBagit(pipeline, workspaceNamespace, workspaceName, bagitRq) pipeTo sender
   }
 
-
   /**
    * Returns the plural form of the entity type.
-   * Bails with a 400 Bad Request if the entity type is unknown to the schema. */
+   * Bails with a 400 Bad Request if the entity type is unknown to the schema and we are using firecloud model
+   * If using flexible model, just appends an 's' */
   private def withPlural(entityType: String)(op: (String => Future[PerRequestMessage])): Future[PerRequestMessage] = {
-    ModelSchema.getPlural(entityType) match {
+    Try(modelSchema.getPlural(entityType)) match {
```

Comment:
This is the type of translation I would expect to see more of as a result of changing the result type of the `ModelSchema` functions to remove the `Try`.